released/discovery_center/mission_4327/setup_subaccount_cf/variables.tf

```diff
@@ -0,0 +1,198 @@
+  type        = list(string)
+  description = "Defines the colleagues who are added to each subaccount as subaccount service administrators."
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+variable "cf_space_managers" {
+  type        = list(string)
+  description = "Defines the colleagues who are Cloudfoundry space managers"
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+variable "cf_space_developers" {
+  type        = list(string)
+  description = "Defines the colleagues who are Cloudfoundry space developers"
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+variable "cf_space_auditors" {
+  type        = list(string)
+  description = "Defines the colleagues who are Cloudfoundry space auditors"
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+variable "cf_org_auditors" {
+  type        = list(string)
+  description = "Defines the colleagues who are Cloudfoundry org auditors"
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+variable "cf_org_managers" {
+  type        = list(string)
+  description = "Defines the colleagues who are Cloudfoundry org auditors"
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+variable "cf_org_billing_managers" {
+  type        = list(string)
+  description = "Defines the colleagues who are Cloudfoundry org auditors"
+  default     = ["jane.doe@test.com", "john.doe@test.com"]
+}
+
+###
+# Entitlements
+###
+variable "entitlements" {
+  type = list(object({
+    service_name = string
+    plan_name    = string
+    type         = string
+  }))
+  description = "The list of entitlements that shall be added to the subaccount."
+  default = [
+    {
+      service_name = "connectivity"
+      plan_name    = "lite",
+      type         = "service"
+    },
+    {
+      service_name = "destination"
+      plan_name    = "lite",
+      type         = "service"
+    },
+    {
+      service_name = "html5-apps-repo"
+      plan_name    = "app-host",
+      type         = "service"
+    },
+    {
+      service_name = "sapappstudio"
+      plan_name    = "standard-edition",
+      type         = "app"
+    },
+    {
+      service_name = "enterprise-messaging"
+      plan_name    = "default",
+      type         = "service"
+    },
+    {
+      service_name = "enterprise-messaging-hub"
+      plan_name    = "standard",
+      type         = "app"
+    },
+    {
+      service_name = "privatelink"
+      plan_name    = "standard",
+      type         = "service"
+    },
+    {
+      service_name = "xsuaa"
+      plan_name    = "application",
+      type         = "service"
+    },
+    {
+      service_name = "hana"
+      plan_name    = "hdi-shared",
+      type         = "service"
+    },
+    {
+      service_name = "hana-cloud"
+      plan_name    = "hana",
+      type         = "service"
+    }
+  ]
+}
+
+
+variable "username" {
+  description = "BTP username"
+  type        = string
+  sensitive   = false
```

Comment:
username as well as password can be supplied via env variables. That is also the preferred way. I would leave them out here